### ISSUE TITLE
Error precompiling webdev

### ISSUE THREAD

**Issue by giorgiofran** [dart-lang/webdev#1429]
I'm using dart 2.13.4.
Yesterday was OK. Today I updated my dependencies (with no error) but, when I run the project, I got some errors errors:

The following is a list of the errors. They seem to be related to the dwds package:
```
Failed to precompile webdev:webdev:
: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
^
: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
^
: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
^
: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
^

: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
^
: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
^
: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
^
: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
^
: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
^
: Error: The specified language version is too high. The highest supported language version is 2.13.
// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
^

Exited (1)

```

**Comment by annagrin** [comment]
@giorgiofran apologies for late reply, I believe this was fixed by publishing webdev 2.7.6. Do you still experience the issue?

**Comment by giorgiofran** [comment]
No, it is working now.
